src/System.Device.Gpio/System/Device/Gpio/GpioDriver.cs

```diff
@@ -82,6 +82,12 @@ protected internal virtual void SetPinMode(int pinNumber, PinMode mode, PinValue
     /// <returns>The value of the pin.</returns>
     protected internal abstract PinValue Read(int pinNumber);
 
+    /// <summary>
+    /// Toggle the current value of a pin.
+    /// </summary>
+    /// <param name="pinNumber">The pin number in the driver's logical numbering scheme.</param>
+    protected internal abstract void Toggle(int pinNumber);
```

Comment:
can this be virtual instead of abstract? (by default Write(!Read())) then concerns about breaking change will be adressed and we don't need to repeat ourselves since most of the drivers don't really natively support it?